source/fundamentals/sessions.txt

```diff
@@ -0,0 +1,154 @@
+.. _golang-sessions:
+
+========
+Sessions
```

Comment:
Should this page just be called sessions and transactions? This page ended up being a lot about transactions, but there were separate tickets for this page and Sessions > Transactions.